app/dao/templates_dao.py

```diff
@@ -33,6 +33,36 @@ def dao_update_template(template):
     db.session.add(template)
 
 
+@transactional
+def dao_update_template_reply_to(template_id, reply_to):
+    Template.query.filter_by(id=template_id).update(
+        {"service_letter_contact_id": reply_to,
+         "updated_at": datetime.utcnow(),
+         "version": Template.version + 1,
+         }
+    )
+    template = Template.query.filter_by(id=template_id).one()
+
+    history = TemplateHistory(**
+                              {
+                                  "id": template.id,
+                                  "name": template.name,
+                                  "template_type": template.template_type,
+                                  "created_at": template.created_at,
+                                  "updated_at": template.updated_at,
+                                  "content": template.content,
+                                  "service_id": template.service_id,
+                                  "subject": template.subject,
+                                  "created_by_id": template.created_by_id,
+                                  "version": template.version,
+                                  "archived": template.archived,
+                                  "process_type": template.process_type,
+                                  "service_letter_contact_id": template.service_letter_contact_id
+                              })
+    db.session.add(history)
+    return history
```

Comment:
Technically do you want to return template here? Although they are very similar?

I know you use it in the test but its the template being updated?

Reply:
I can return either - makes more sense to use template. I'll update